packages/p2p/src/components/buy-sell/buy-sell-modal.scss

```
@@ -39,6 +39,22 @@
         }
     }
 
+    &-hintbox {
+        margin: -2.4rem 0 2.4rem;
+
+        @include mobile {
+            margin: 0 0 2.4rem;
+        }
+
+        .dc-hint-box {
+            &__icon {
```

```suggestion
        .dc-hint-box > &__icon {
```
Just a nit, since there's only one level of nesting using the `>` child operator makes it cleaner

---

It's not working for mobile view, styles are broken if we use this way

---

👍🏼